analysers/analyser_osmosis_highway_tunnel_bridge.py

```diff
@@ -25,20 +25,26 @@
 
 sql10 = """
 SELECT
-    id,
-    ST_AsText(way_locate(linestring))
+    ways.id,
+    ST_AsText(way_locate(ways.linestring))
 FROM
-    {0}ways
+    {0}ways AS ways
+    LEFT JOIN {1}polygons AS contours ON
+        ST_Intersects(contours.poly, ways.linestring) AND
```

Comment:
But you implemented it with an `ST_Intersects` and not a `ST_WithIn` ?

Reply:
Yes, I think a single road can pass through multiple man_made=bridge sections, e.g. half arch bridge, half suspension bridge. So ST_WithIn wouldn't catch these. But the wiki isn't fully clear to me about this, and I guess these cases are rare...

On the other hand, ST_Intersects also has its flaws (see the footnote in the original post).

You can choose: which one would you prefer?

Reply:
I've just changed it to the more restrictive case. This covers at least the vast majority of the cases